README.md

```diff
@@ -143,6 +143,10 @@ it will have the following arguments:
   example `./pkg/github-client/options_generated.go`.
 - `all-variadic` - generate variadic functions for all fields with slice type.
   Default: `false` - functions that accept a slice are generated.
+- `generate-constructor` - flag for whether to generate a function to build your structure with options.
```

Comment:
Could you please rename it to `with-constructor` like `with-isset`?

Reply:
Motivation: 1) similar way to enable feature; 2) a bit shorter options (if you can find a shorted but clean word - please use it)

Reply:
Probably we can re-invent the flags:

```
-constructor=public // Default behaviour that works right now.
-constructor=no // Disable any constructor
-constructor=private // Generate private constructor
-
```